iceoryx_hoofs/test/moduletests/test_unix_domain_sockets.cpp

```diff
@@ -78,36 +125,229 @@ class UnixDomainSocket_test : public Test
 const size_t UnixDomainSocket_test::MaxMsgSize = UnixDomainSocket::MAX_MESSAGE_SIZE;
 constexpr uint64_t UnixDomainSocket_test::MaxMsgNumber;
 
-TEST_F(UnixDomainSocket_test, NonBlockingModeNotSupported)
+TEST_F(UnixDomainSocket_test, UnlinkEmptySocketNameLeadsToInvalidChannelNameError)
```

Comment:
Aren't this tests for all IPC channel implementations like the message queue, in order to ensure similar behavior?

Reply:
The unlink checks are checks for the ipc channel but were not really done there. Therefore, I created them here specific for the unix domain socket where I created sockets etc.

The unix domain socket also offers features which are not used by the IPC channel and I wanted to clearly test them in a unit test.